src/generators/php/index.js

```diff
@@ -143,12 +143,9 @@ class PHP extends Generator {
       default:
         if (enumMap[typeName]) {
           if (this.includedInSchema(enumMap[typeName].namespace)) {
-            return "Schema.NET." + this.convertToCamelCase(typeName);
+            return "\\OpenActive\\Enums\\" + this.convertToCamelCase(typeName);
```

Comment:
if clause is not doing anything here - always just return `return "\\OpenActive\\Enums\\" + this.convertToCamelCase(typeName);`

Reply:
Yeah I thought of leaving it in as a memo that we need to fix schema.org classes - but happy to remove otherwise :)

Reply:
Also I saw you fixing it in models-php, happy to merge there anyway (unless you've deleted it).

Reply:
I think in this case use a comment about how OA and Schema.org enums are stored in the same namespace in the PHP currently (unlike the models) rather than having a dubious if-else.

Reply:
I've deleted my work on the same issue